src/options/components/JsonConfig.js

```diff
@@ -0,0 +1,87 @@
+import React, { useRef, useState } from "react";
+import PropTypes from "prop-types";
+import parserBabylon from "prettier/parser-babylon";
+import prettier from "prettier/standalone";
+
+const PLACEHOLDER_TEXT = `{
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2
+}`;
```

Comment:
Placeholder text is a good idea, but maybe it would be more useful to display the default options.

Reply:
Sure, that sounds good to me

Reply:
We don't need a placeholder any more because we're always displaying what the user has configured.